src/platforms/dotnet/migration.mdx

```diff
@@ -108,3 +108,183 @@ A new package was added ASP.NET Core gPRC support. With the ability to log the r
 # Release notes
 
 Please refer to [GitHub for the complete release notes](https://github.com/getsentry/sentry-dotnet/releases/tag/3.0.0).
+
+<Note>
+
+Migrating from SharpRaven to Sentry SDK
+
+</Note>
+
+# Breaking changes
+
+## .NET Framework support reduced
+
+If your project is targeting .NET Framework, it will need to be using at least v4.6.1 to use Sentry (SharpRaven supported .NET Framework v3.5 and higher).
+
+## Initialization
+
+Changed how to initialize Sentry:
+
+```csharp
+using Sentry;
+
+// Initialize with DSN
+SentrySdk.Init("___PUBLIC_DSN___");
+
+// Initialize with options
+SentrySdk.Init(o =>
+{
+    o.Dsn = "___PUBLIC_DSN___";
+});
+```
+
+For ASP.NET (classic) integration, add `Sentry.AspNet` package and initialize it by calling `o.AddAspNet()`:
+
+```csharp
+using Sentry;
+using Sentry.AspNet;
+
+SentrySdk.Init(o =>
+{
+    o.Dsn = "___PUBLIC_DSN___";
+
+    o.AddAspNet();
+});
+```
+
+## Event queue
+
+Contrary to SharpRaven, events in Sentry SDK are sent in background without blocking the currently executing code. Calling `SentrySdk.CaptureException(...)` does not immediately send an event, but instead queues it up on a background thread.
+
+Usage changes:
+
+```csharp
+// Before
+ravenClient.Capture(...);
+await ravenClient.CaptureAsync(...);
+
+// After
+SentrySdk.CaptureEvent(...);
+```
+
+You can flush the event queue by invoking the `IDisposable` returned by `SentrySdk.Init(...)`:
+
+```csharp
+var sentryInitialization = SentrySdk.Init("___PUBLIC_DSN___");
+
+// Capture a non-error message
+SentrySdk.CaptureMessage("hello world");
+
+// Tear down Sentry and flush queued up events
+sentryInitialization.Dispose();
+```
+
+## Event processors
+
+Usage changes:
+
+```csharp
+// Before
+ravenClient.BeforeSend = requester =>
+{
+    // Here you can log data from the requester
+    // or replace it entirely if you want.
+    return requester;
+};
+
+// After
+SentrySdk.Init(o =>
+{
+    o.BeforeSend = e =>
+    {
+        // Mutate the event or return null to drop it altogether
+        return e;
+    };
+}
+});
+```
+
+## List of differences between SharpRaven and SentrySdk
```

Comment:
This list is super useful - I suggest it'd be better to list it prior to the breaking changes to help folks understand the value of migrating.